static/css/section/_server.scss

```diff
@@ -1,213 +1,108 @@
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Server section
+//  Author:			Web Team at Canonical Ltd
+//
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
 
-body.server {
+// @global server
+.server {
 
-  .row-hero { /* First bit of content on a page, normally contains h1 and sometimes a bit of blurb */
+  .row-hero {
     position: relative;
-
-    h2 {
-      color: $warm-grey;
-      padding-top: 20px;
-    }
-  }
-
-  tfoot th {
-    font-weight: bold;
-  }
-
-  .button-right {
-    margin-top: 18px;
-    margin-bottom: 0;
-  }
-
-  .row-landscape img {
-    height: auto;
-    max-width: 100%;
   }
 
-  .row-community {
-    margin-bottom: 0;
-    border-bottom: 0;
-    padding: 40px 20px;
-  }
+  .box-padded-feature {
+    @media only screen and (min-width : $breakpoint-medium) {
+      .six-col {
+        border-right: 1px dotted $warm-grey;
+      }
 
-  .row-community .nine-col {
-    background: #fff;
-  }
+      .six-col.last-col {
+        border-right: 0;
+      }
 
-  .box-padded ul.inline-icons {
-    margin-bottom: 0;
+      .border-bottom {
+        border-bottom: 1px dotted $warm-grey;
+        margin-bottom: 10px;
+        padding-bottom: 10px;
+      }
+    }
   }
 }
 
-.row-juju-maas div {
-  padding-bottom: 0;
-}
-
-/*	@section server-services
--------------------------------------------------------------- */
-body.server-services .row-hero {
-  background-image: none;
-  border: 0;
-  padding-bottom: 0;
-}
-
-body.server-services .row-hero h2 {
-  background: none;
-  color: inherit;
-  padding: 0;
-  text-align: left;
-}
-
-body.server-services .packages {
-  margin-bottom: 20px;
-}
-
-body.server-services .packages h2 {
-  background: #efeeec url('#{$asset-server}87667959-bg-servicesserver-owl-125x101.png') no-repeat 20px 50%;
-  width: 45.5%;
-  padding-top: 85px;
-  padding-bottom: 77px;
-}
-
-body.server-services .packages dd {
-  min-height: 10em;
-}
-
-body.server-services .packages dd.price {
-  min-height: 0;
-}
-
-body.server-services .packages .price-annual  {
-  top: 220px;
-}
-
-body.server-services .row-ubuntu-advantage {
-  padding-bottom: 0;
-}
-
-body.server-services .row-ubuntu-advantage h3 {
-  font-size: 1.21875em;
-}
-
-body.server-services .row-ubuntu-advantage .list-details {
-  margin-bottom: 0;
-}
+// @section /server overview
+.server-home {
 
-body.server-services .row-ubuntu-advantage li.first {
-  height: 38.7em;
-}
-
-body.server-services .row-blockquote {
-  background: #e2d4dc;
-  padding-top: $gutter-width;
-  margin-bottom: 0;
-  padding-bottom: $gutter-width/2;
-}
-
-body.server-services .row-blockquote p.last {
-  padding-left: 25px;
-  display: block;
-  float: left;
+  .row-juju-maas div {
+    padding-bottom: 0;
```

Comment:
Please remove as its causing no padding on mobile and works ok without on desktop